app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -652,43 +652,70 @@ class AdvancedFilter extends React.Component {
    */
   getRelativeTooltipString(filter, value) {
     const filterName = filter.title.replace(' (Relative Date)', '');
-    let statement, rangeString, start, end;
+    let rangeString, before, after;
+    let statement = '';
+    const operatorValue = value.operator.replace('-', ' ');
 
-    // set variables for date options including a time stamp
-    if (filter.hasTimestamp) {
+    if (value.operator === 'more-than') {
       if (value.when === 'past') {
-        rangeString = 'dated through today’s date';
-        start = moment()
+        before = moment()
           .subtract(value.number, value.unit)
           .format('MM/DD/YY');
-        end = 'now';
       } else {
-        rangeString = 'with today’s date as of the current time';
-        start = 'now';
-        end = moment()
+        after = moment()
           .add(value.number, value.unit)
           .format('MM/DD/YY');
       }
+    } else if (value.operator === 'less-than') {
+      // set variables for date options including a time stamp
+      if (filter.hasTimestamp) {
+        if (value.when === 'past') {
+          rangeString = 'dated through today’s date';
+          after = moment()
+            .subtract(value.number, value.unit)
+            .format('MM/DD/YY');
+          before = 'now';
+        } else {
+          rangeString = 'with today’s date as of the current time';
+          after = 'now';
+          before = moment()
+            .add(value.number, value.unit)
+            .format('MM/DD/YY');
+        }
+      }
+
+      // set variables for date options without a timestamp
+      else {
+        if (value.when === 'past') {
+          rangeString = 'dated through today’s date';
+          after = moment()
+            .subtract(value.number, value.unit)
+            .format('MM/DD/YY');
+          before = moment().format('MM/DD/YY');
+        } else {
+          rangeString = 'with today’s date';
+          after = moment().format('MM/DD/YY');
+          before = moment()
+            .add(value.number, value.unit)
+            .format('MM/DD/YY');
+        }
+      }
     }
 
-    // set variables for date options without a timestamp
-    else {
+    if (value.operator === 'less-than') {
+      statement = `${filterName} “${operatorValue} ${value.when}” relative date periods include records ${rangeString}. `;
+    }
+    statement += `The current setting of "${operatorValue} ${value.number} ${value.unit} in the ${value.when}" will return records with ${filterName} date`;
+    if (value.operator === 'less-than') {
+      statement += ` from ${after} through ${before}. `;
+    } else {
       if (value.when === 'past') {
-        rangeString = 'dated through today’s date';
-        start = moment()
-          .subtract(value.number, value.unit)
-          .format('MM/DD/YY');
-        end = moment().format('MM/DD/YY');
+        statement += ` before ${before}. `;
```

Comment:
Tooltip wording here is incorrect because the current date IS included (because of [this](https://github.com/SaraAlert/SaraAlert/blob/master/app/helpers/patient_query_helper.rb#L498) line: `patients = patients.where('symptom_onset <= ?', before)`) but the tooltip says otherwise.
![Screen Shot 2021-02-17 at 1 18 12 PM](https://user-images.githubusercontent.com/11698457/108249266-db7a8a80-7122-11eb-82e8-9ae06cd86e7c.png)

Reply:
This is actually correct because of this:
```
when 'more-than'
  timeframe = { before: (timespan.ago - tz_diff).beginning_of_day } if filter[:value][:when] == 'past'
  timeframe = { after: (timespan.from_now - tz_diff).end_of_day } if filter[:value][:when] == 'future'
end
```

Reply:
If you applied the filter in the screenshot, you would see that 2/10/21 is not included in the results but 2/9/21 is

Reply:
I had applied the filter, and it is included which was what spurred the original comment haha

Reply:
https://user-images.githubusercontent.com/35042815/108260389-47172480-7130-11eb-8601-e009850de84e.mov

I'm seeing the tooltip match the behavior of the filter (see video).  Can you give me steps to recreate what you're seeing?

Reply:
1. Update a record in the Isolation workflow to have a symptom onset of 1 week ago (`p.update!(symptom_onset: 1.week.ago)`)
2. Create filter for relative symptom onset of more than 1 week in the past.
3. Notice that results includes the record with a symptom onset date that is exactly 1 week in the past (2/10/21)
4. Notice that the tooltip says it will only include results before 2/10/21 which is incorrect

https://user-images.githubusercontent.com/11698457/108262052-6d3dc400-7132-11eb-8786-9e1ebdc1d7e9.mov

Reply:
1. Update a record in the Isolation workflow to have a symptom onset of 1 week ago (p.update!(symptom_onset: 1.week.ago))
2. Create filter for relative symptom onset of more than 1 week in the past.
3. Notice that results includes the record with a symptom onset date that is exactly 1 week in the past (2/10/21)
4. Notice that the tooltip says it will only include results before 2/10/21 which is incorrect

https://user-images.githubusercontent.com/11698457/108262348-cf96c480-7132-11eb-969c-db8829407798.mov

Reply:
Ah I see the issue with a field with no timestamp.  Clarified the tooltip